sdk/docs/pages/typescript-sdk/hello-sui.mdx

```diff
@@ -0,0 +1,86 @@
+# Hello Sui
+
+import { Callout } from 'nextra/components';
+
+
+
+This basic example introduces you to the Sui TypeScript SDK. The Node.js example mints SUI on Testnet and then queries the address to get a sum for the owned SUI. You don't need to use an IDE to complete the example, but one like Microsoft Visual Studio Code helps centralize more advanced projects. 
+
+You need [Node.js](https://nodejs.org/en/download/current) and [pnpm](https://pnpm.io/installation) to follow this example, so install them on your system if you haven't already. 
+
+## Before you begin
+
+You need an address on a Sui development network (Devnet, Testnet, Localnet). If you don't already have an address, use the [Sui Client CLI](https://docs.sui.io/build/cli-client) or the [Sui Wallet browser extension](https://docs.mystenlabs.com/wallet) to create one. 
+
+## Clone Sui project and build SDK
```

Comment:
I'm not sure we need this as part of the docs site, at least not as a prominent part. 

Generally, I think we want people to use the published version of the client rather than building/linking a local version of the SDK